### PR TITLE
Privacy policy page with smart scroll

### LINKED ISSUE #50: Create UI for privacy page
## UI Design for privacy page

Here is the final design. Please refer to screenshot below.
 
evalai_redesign_privacy_final_updated

---

Please comment below if you want to work on this or in case of any question/suggestion.

@shivaniprakash95  please share the details about `assets` and `content` that will be needed for this.

### PR DESCRIPTION
**Pull request by live-wire**
Privacy policy page available at path `/privacy-policy`
Supposed to close #50 
- Implemented a smart scroll navigation for this page
(I removed this scroll-nav functionality for mobile view as it looks too cluttered).

> The header will look better once #61 is merged

### CODE REVIEW

File: src/app/privacy-policy/privacy-policy.component.scss [hunk 1]
```diff
@@ -0,0 +1,84 @@
+@import '../../styles/base.scss';
```
Comment (aka-jain) [inline src/app/privacy-policy/privacy-policy.component.scss]: I think you don't have to import `base` and `grid` again and again  @live-wire

File: src/app/privacy-policy/privacy-policy.component.ts [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+    [].forEach.call(allNavs, function (item) {
+      item.classList.remove('scroll-selected');
+      if (item.id === el) {
+        item.className += ' scroll-selected';
+      }
+    });
+  }
+  ngOnInit() {
+    this.document['manuallyScrolling'] = false;
+    const idText = 'privacy-';
```
Comment (aka-jain) [inline src/app/privacy-policy/privacy-policy.component.ts]: for const use variable in capital letters eg. const IDTEXT @live-wire

**Review by aka-jain — CHANGES_REQUESTED** [review]
What is smart scrolling @live-wire can you please explain here.

**Comment by live-wire** [comment]
*Smart scrolling* is just a term I came up with 🙈 
There is a navigation-indicator on the left which shows what section we're currently at in the privacy policy. It has the following features:
- Highlight the nav element for the selected section (red).
- Links to the corresponding section and scrolls it into view on click.
- Automatically updates the selected section when the page is manually scrolled.
- Disappears in mobile-view (It was too cluttered)

Elements in the *nav* are decided by checking for class: `privacy-nav`
Elements in the corrsponding titles are decided by checking for class: `privacy-section-title`

**Comment by live-wire** [comment]
Hey @aka-jain, I updated this PR with the suggested changes. Please have a look. 🎈

File: src/app/privacy-policy/privacy-policy.component.html [hunk 3]
```diff
@@ -0,0 +1,279 @@
+<app-header-static></app-header-static>
+<div class="container-top container privacy-container">
+<div class="privacy-nav-container col-lg-2 col-md-3 col-sm-5 col-xs-5">
```
Comment (aka-jain) [inline src/app/privacy-policy/privacy-policy.component.html]: Wrap col under row @live-wire

File: src/styles/variables.scss [hunk 4]
```diff
@@ -23,3 +30,8 @@ $fs-20: 20px;
 $fs-22: 22px;
 $fs-24: 24px;
 $fs-36: 36px;
+
+/*Screen Sizes*/
+$screen-sm: 810px;
+$screen-sm-md: 992px;
+$screen-md: 1130px;
```
Comment (aka-jain) [inline src/styles/variables.scss]: New line @live-wire

**Review by aka-jain — CHANGES_REQUESTED** [review]

**Comment by live-wire** [comment]
Made the suggested changes @aka-jain 🏃

**Comment by live-wire** [comment]
jun-4-2018-11_26-pm
☝️ This is how the `/privacy-policy` page looks like

Please have a look @spyshiv @aka-jain @RishabhJain2018 🍬 

cc: @shivaniprakash95

**Comment by RishabhJain2018** [comment]
I have tested it & it works fine.

**Comment by RishabhJain2018** [comment]
@live-wire Can you please check why travis build fails?

**Comment by RishabhJain2018** [comment]
@live-wire Please resolve the merge conflicts.

**Comment by RishabhJain2018** [comment]
@live-wire More merge conflicts 😄

**Comment by live-wire** [comment]
@RishabhJain2018 resolved them conflicts :D

**Review by RishabhJain2018 — APPROVED** [review]
LGTM :)
@aka-jain @spyshiv Can we merge this PR ?

**Review by aka-jain — APPROVED** [review]

**Pull request merged**